universal-application-tool-0.0.1/app/auth/CiviFormProfile.java

```diff
@@ -62,30 +64,54 @@ public CiviFormProfile(
         dbContext);
   }
 
+  /**
+   * Get the client name associated with the profile.
+   *
+   * <p>This is the name set for the client object, more close to a class name, not the name of a
```

Comment:
Instead of this sentence I'd say "this is the class name of the pac4j client used to create this profile."

Reply:
Thanks! Done.